locale/core.yml

```diff
@@ -238,8 +238,11 @@ core:
 
     # These translations are used in the Edit User modal dialog (admin function).
     edit_user:
+      activate: Activate User
       email_label: => core.ref.email
+      groups_label: => Groups
       password_label: => core.ref.password
+      set_new_password: Set new password
```

Comment:
This key name also needs a suffix ... to keep it short, I'd suggest `set_password_label`.